src/ui/map.js

```diff
@@ -160,6 +160,7 @@ const defaultOptions = {
     touchZoomRotate: true,
     touchPitch: true,
     cooperativeGestures: false,
+    enablePerformanceMetricsCollection: true,
```

Comment:
nit: maybe rename to `performanceMetricsCollection: true`. `enable` is implicit for all boolean map constructor options